scripts/release_scripts/initial_release_prep.py

```diff
@@ -0,0 +1,321 @@
+            instruction_doc.
+
+    Returns:
+        string. The mail message template.
+    """
+    job_names_with_instructions_and_author = ''
+    author_names_and_mail_ids = []
+    for job_detail in job_details:
+        job_names_with_instructions_and_author += (
+            '%s (Instructions: %s) (Author: %s)\n' % (
+                job_detail['job_name'], job_detail['instruction_doc'],
+                job_detail['author_name']))
+        author_names_and_mail_ids.append(
+            '%s: %s' % (
+                job_detail['author_name'], job_detail['author_email']))
+    author_names_and_mail_ids = list(set(author_names_and_mail_ids))
+    mail_message_template = (
+        'Hi Sean,\n\n'
+        'You will need to run these jobs on the backup server:\n\n'
+        '%s\n'
+        'The specific instructions for jobs are linked with them. '
+        'The general instructions are as follows:\n\n'
+        '1. Login as admin\n'
+        '2. Navigate to the admin panel and then the jobs tab\n'
+        '3. Run the above jobs\n'
+        '4. In case of failure/success, please send the output logs for '
+        'the job to me and the job authors: %s\n\n'
+        'Thanks!\n') % (
+            job_names_with_instructions_and_author, author_names_and_mail_ids)
+    return mail_message_template
+
+
+def get_extra_jobs_due_to_schema_changes(
+        remote_alias, previous_release_version):
+    """Finds additional jobs which should be run based on
+    schema changes in feconf.
+
+    Args:
+        remote_alias: str. The alias for Oppia repo.
+        previous_release_version: str. The version of the previous release.
+
+    Returns:
+        list(str). The list of jobs to run based on schema changes.
+    """
+    schema_versions_to_jobs_mapping = {
+        'CURRENT_COLLECTION_SCHEMA_VERSION': 'CollectionMigrationOneOffJob',
+        'CURRENT_STATE_SCHEMA_VERSION': 'ExplorationMigrationJobManager',
+        'CURRENT_SKILL_CONTENTS_SCHEMA_VERSION': 'SkillMigrationOneOffJob',
+        'CURRENT_MISCONCEPTIONS_SCHEMA_VERSION': 'SkillMigrationOneOffJob',
+        'CURRENT_RUBRIC_SCHEMA_VERSION': 'SkillMigrationOneOffJob',
+        'CURRENT_STORY_CONTENTS_SCHEMA_VERSION': 'StoryMigrationOneOffJob',
+        'CURRENT_SUBTOPIC_SCHEMA_VERSION': 'TopicMigrationOneOffJob',
+        'CURRENT_STORY_REFERENCE_SCHEMA_VERSION': 'TopicMigrationOneOffJob'}
+
+    diff_output = common.run_cmd([
+        'git', 'diff', '%s/develop' % remote_alias,
+        '%s/release-%s' % (remote_alias, previous_release_version),
+        'feconf.py'])
+    feconf_diff = diff_output[:-1].split('\n')
+
+    jobs_to_run = []
+    for version_key in schema_versions_to_jobs_mapping:
+        for line in feconf_diff:
+            if line.startswith(('+%s' % version_key, '-%s' % version_key)):
+                jobs_to_run.append(schema_versions_to_jobs_mapping[version_key])
+
+    return list(set(jobs_to_run))
+
+
+def check_changes_in_supported_audio_languages(
```

Comment:
did_supported_audio_languages_change() or have_supported_audio_languages_changed() 

(to imply that a boolean is being returned)

Reply:
Done.